StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsAuthSession.swift

```diff
@@ -76,6 +76,7 @@ struct FinancialConnectionsAuthSession: Decodable {
     let skipAccountSelection: Bool?
     let url: String?
     let isOauth: Bool?
+    let display: Display?
```

Comment:
backend link for double check:
- https://git.corp.stripe.com/stripe-internal/pay-server/pull/557916/files#diff-2870239e3af12e5f749b97082cd6d88865392915c5a7e5cccf87f57a14fcee29R24